include/rocksdb/statistics.h

```diff
@@ -337,6 +337,18 @@ enum Tickers : uint32_t {
   // # of bytes in the blob files evicted because of BlobDB is full. Only
   // applicable to legacy BlobDB.
   BLOB_DB_FIFO_BYTES_EVICTED,
+  // # of times cache miss when accessing blob from blob cache.
+  BLOB_DB_CACHE_MISS,
+  // # of times cache hit when accessing blob from blob cache.
+  BLOB_DB_CACHE_HIT,
+  // # of data blocks added to blob cache.
+  BLOB_DB_CACHE_ADD,
+  // # of failures when adding blobs to blob cache.
+  BLOB_DB_CACHE_ADD_FAILURES,
```

Comment:
Since these are enums, won't this cause any compatibility issue cc @ltamasi? Tests seems to be passing but in the comments its mentioned "Any ticker should be added immediately before TICKER_ENUM_MAX, taking
 *     over its old value."

Reply:
Oops! Thanks, @akankshamahajan15  I think you are right.

Reply:
It seems that the comment doesn't match the actual behavior.

Reply:
It's hard to say these "BLOB_DB_*" were added at the same time. Some of them are related to "Stack" BlobDB, others are Integrated BlobDB.

Reply:
We reused the stacked BlobDB's tickers for the integrated implementation wherever appropriate and deprecated the ones that didn't apply to the new design.

Reply:
done